src/Zypper.cc

```diff
@@ -3200,6 +3201,8 @@ void Zypper::processCommandOptions()
 	     _("Create a short table not showing the deleted files. Given twice, show only processes which are associated with a system service. Given three times, list the associated system service names only.") )
     .option( "--print <format>",	// translators: --print <format>
 	     _("For each associated system service print <format> on the standard output, followed by a newline. Any '%s' directive in <format> is replaced by the system service name.") )
+    .option("-d, --debugFile <path>",
+       _("Write debug output to file <path>."))
```

Comment:
Precede translatable option description with `// translators: -d, --debugFile <path>`, like in the lines above.

Comments preceding a translatable string go into the .pot file. Hints for the translator are tagged as `// translators: `

Reply:
Done

Reply:
Sorry, I forgot: Please update the man page accordingly.